cmd/snapd-apparmor/main.go

```diff
@@ -44,18 +44,11 @@ import (
 
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/logger"
+	"github.com/snapcore/snapd/release"
 	apparmor_sandbox "github.com/snapcore/snapd/sandbox/apparmor"
 	"github.com/snapcore/snapd/snapdtool"
 )
 
-func isWSL() bool {
-	if output, err := exec.Command("systemd-detect-virt", "--container").Output(); err == nil {
-		virt := strings.TrimSpace(string(output))
-		return virt == "wsl"
```

Comment:
Out of curiosity, what does "systemd-detect-virl"/"systemd-detect-virt --container" output on a WSL1 and WSL2 system? And is that the same?

Reply:
Yes, it prints "wsl" in all cases.